deploy/index.html

```diff
@@ -37,7 +37,7 @@
         <div class="col-sm-8">
         <p>
           <img src="/images/pages/deploy/duke.jpg" width="100" height="80" class="pull-right" alt="Duke">
-          <a href="http://jruby.codehaus.org/" title="The Ruby Programming Language on the JVM">JRuby</a> brings Rails to the Java Virtual Machine. This means that you can deploy Rails applications on app servers like <a href="https://glassfish.dev.java.net/" title="Java EE 7 Application server">Glassfish</a> or <a href="http://jetty-rails.rubyforge.org/" title=" Jetty Rails: Java Web server">Jetty</a>. You can use <a href="http://caldersphere.rubyforge.org/warbler/" title="Warbler gem">Warbler</a> to package your Rails application as a standard WAR. Great for slipping into the enterprise.
```

Comment:
yep - on Github, most recent commit is from 2009 (https://github.com/fabiokung/jetty-rails)